src/status_im2/contexts/communities/discover/view.cljs

```diff
@@ -206,15 +206,16 @@
         featured-communities-count (count featured-communities)]
     (fn []
       [scroll-page/scroll-page
-       {:theme          theme
-        :on-scroll      #(reset! scroll-height %)
-        :navigate-back? :true
-        :height         (if (> @scroll-height 360)
-                          208
-                          148)
-        :sticky-header  [render-sticky-header
-                         {:selected-tab  selected-tab
-                          :scroll-height scroll-height}]}
+       {:theme            theme
+        :on-scroll        #(reset! scroll-height %)
+        :navigate-back?   :true
+        :height           (if (> @scroll-height 360)
+                            208
+                            148)
+        :background-color (colors/theme-colors colors/white colors/neutral-95)
```

Comment:
FYI: Fix from past regression, here I'm just bringing back the missing background color.